WorkingInR/2_First_Steps_in_R.Rmd

```diff
@@ -363,31 +393,40 @@ big.list
 
 ## Describing an object
 
-Further below you will find a handful of functions that help you analyze objects. Some of these functions will tell you how the data stored inside the object look like, others will give you an idea of its structure, the data type(s) or simply a quick overview of the data set.
+Further below you will find a handful of functions that help you analyze objects. 
+Some of these functions will tell you how the data stored inside the object look like, others will give you an idea of its structure, the data type(s) or simply a quick overview of the data set.
+When working with tibbles, most of these functions are not of much interest or use since R always describes the structure of the tibble when printing it. 
+However when working with data frames, these functions will give you a better overview.
 
-### Looking at the Environment tab
+
+### The Environment tab
 
```

Comment:
In my opinion we do not need the env tab here. It should be introduced in the RStudio intro part. But I am happy to give in if you think it is useful to show it here again.